cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -309,7 +309,43 @@ resource {{bicepName $name}}RoleAssignment 'Microsoft.Authorization/roleAssignme
   }
 }
 {{- end}}
-
+{{range $name, $value := .CosmosDbAccounts}}
+resource {{bicepName $name}} 'Microsoft.DocumentDB/databaseAccounts@2023-11-15' = {
+  name: replace('{{$name}}-${resourceToken}', '-', '')
+  location: location
+  tags: union(tags, {'aspire-resource-name': '{{$name}}'})
+  properties: {
+    consistencyPolicy: { defaultConsistencyLevel: 'Session' }
+    locations: [
+      {
+        locationName: location
+        failoverPriority: 0
+      }
+    ]
+    databaseAccountOfferType: 'Standard'
+  }
+}
+// Cosmos DB Account Reader Role
+resource {{bicepName $name}}ReaderRoleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```

Comment:
These permissions are related to the control-plane, not the data plane.  Do we also need to do anything to grant the app access to data plane operations via RBAC?

Reply:
We can likely use our existing Cosmos templates as a starting point template for aspire and modify from there.

Reply:
@ellismg , I am not sure. It all depends on what the application is going to do...
- Will it create/admin dbs?
- Will it only read documents?
- Will it read/write?

Depending on that, we might need to add some other roles in the future. Or if there is no a role for data-plane, we might need to switch from RBAC to connectionString+key :(

Reply:
I think that out of the box we'll want to be permissive in the roles we grant to the application (so it can do both management and data plane operations).  This way things "just work" out of the box.  Folks who want to control the permissions can use `infra synth` and then change the RBAC permissions if they want.

When we think about configuration knobs for the infrastructure long term, maybe allow the user to change the role we assign the worker is something we can support?

I do think that we want read/write support out of the box so sample apps like blogs and the like just work.